package.json

```diff
@@ -9,7 +9,7 @@
   ],
   "scripts": {
     "develop": "gatsby develop",
-    "start": "gatsby develop",
+    "start": "gatsby develop --host=0.0.0.0",
```

Comment:
How does this æffect running it in production? (if at all) @niklasmtj

Reply:
This shouldn't effect it at all since we're pushing the public folder to GH pages